relay-replays/src/snapshots/relay_replays__recording__tests__scrub_pii_key_based_edge_cases.snap

```diff
@@ -0,0 +1,45 @@
+---
+source: relay-replays/src/recording.rs
+expression: scrubbed
+---
+[
+  {
+    "data": {
+      "payload": {
+        "data": {
+          "method": "POST [ip]",
```

Comment:
An IP appearing in `method` is scrubbed, because `data.payload.data` is in `PII_FIELDS`.